src/main/java/it/pagopa/transactions/services/v1/TransactionsService.java

```diff
@@ -451,7 +451,6 @@ public Mono<Void> cancelTransaction(String transactionId, UUID xUserId) {
 
     }
 
-    @CircuitBreaker(name = "transactions-backend")
     @Retry(name = "requestTransactionAuthorization")
```

Comment:
what about remove also retry handler here? for 5xx error codes retry will already be performed by f.e.
What do you think?

Reply:
We decide to let the retry here for the time we will switch from PGS to NPG and back